azkaban-common/src/main/java/azkaban/imagemgmt/models/ImageOwnership.java

```diff
@@ -15,13 +15,16 @@
  */
 package azkaban.imagemgmt.models;
 
+import com.google.common.collect.ImmutableMap;
+import java.util.Arrays;
 import javax.validation.constraints.NotBlank;
 import org.codehaus.jackson.annotate.JsonProperty;
 
 /**
  * This class represents ownership information of an image type.
  */
 public class ImageOwnership extends BaseModel {
```

Comment:
One question: Why are you not using lombok annotations for getter and setter?

Reply:
As suggested by Java team in the wiki - https://iwww.corp.linkedin.com/wiki/cf/display/ENGS/Java+Coding+Conventions?focusedCommentId=300521601#comment-300521601
there are some run time  dependency issues as the artifacts are maintained by the foundation team and lombok depends on byte code manipulation. Eventhough few folks don't aggree, as suggested by foundation team better to avoid using lombok .